code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -618,14 +621,15 @@
 	var/key = get_pin_data(IC_INPUT, 4)
 
 	var/datum/netdata/data = new
-	data.recipient_ids += target_address
+	data.recipient_ids = splittext(target_address, ";")
+	data.sender_id = address
 	data.plaintext_data = message
 	data.plaintext_data_secondary = text
 	data.plaintext_passkey = key
 	ntnet_send(data)
 
 /obj/item/integrated_circuit/input/ntnet_recieve(datum/netdata/data)
```

Comment:
you might want to move this to 
/obj/item/integrated_circuit/input/ntnet_packet/ntnet_recieve

I tested it and it works

Reply:
How I will move sending information about sender to recieving proc?